docs/vector/README.md

```diff
@@ -0,0 +1,161 @@
+# Using Vector with k8ssandra-operator
+
+Since k8ssandra-operator v1.5.0, it is possible to scrape Cassandra metrics with the Vector agent and customize its configuration to add transformers and sinks.
+
+More information about Vector can be found in the [official documentation](https://vector.dev/docs/).
+
+## Enabling Vector agent
+
+To enable Vector agent, you need to add a `.spec.cassandra.telemetry` section in the `K8ssandraCluster` manifest:
+
+```yaml
+apiVersion: k8ssandra.io/v1alpha1
+kind: K8ssandraCluster
+metadata:
+  name: test
+spec:
+  cassandra:
+    serverVersion: 4.0.6
+    telemetry:
+      vector:
+        enabled: true
+        resources:
+          requests:
+            cpu: 100m
+            memory: 128Mi
+          limits:
+            cpu: 1000m
+            memory: 1Gi
+...
+...
+```
+
+The following content will be added automatically to the vector.toml file, setting up Cassandra metrics scraping:
+
+```toml
+data_dir = "/var/lib/vector"
+
+[api]
+enabled = false
+  
+[sources.cassandra_metrics]
+type = "prometheus_scrape"
+endpoints = [ "http://localhost:{{ .ScrapePort }}" ]
+scrape_interval_secs = {{ .ScrapeInterval }}
+```
+
+The `config` section allows you to specify a `ConfigMap` containing a custom Vector configuration. If not specified, the default configuration will be used and the metrics will be logged in the console output of the vector container:
+
+```toml
+[sinks.console]
+type = "console"
+inputs = [ "cassandra_metrics" ]
+target = "stdout"
+
+  [sinks.console.encoding]
+  codec = "json"`
+```
+
+
+## Custom Vector configuration
+
+To customize the Vector configuration, you need to create a `ConfigMap` containing a `vector.toml` file. The `ConfigMap` must be in the same namespace as the `K8ssandraCluster` resource:
```

Comment:
From what I see in the `reconcileVector` method, it needs to be in the same namespace as the datacenter, which in turn defaults to the `K8ssandraCluster` namespace. But these can be different in a multi-namespace deployment.